P/Pinchot/build_tarballs.jl

```diff
@@ -0,0 +1,39 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "Pinchot"
+version = v"13.0.1"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/joeynelson/pinchot-c-api.git", "54cb91757513145b3bf3358be9d2818095b128ca")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/pinchot-c-api
+mkdir build
+cd build/
+cmake .. -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release -DBUILD_SHARED_LIBS=ON
+make
+make install
+install_license ${WORKSPACE}/srcdir/pinchot-c-api/LICENSE.txt
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
```

Comment:
Let's try to build for all platforms, including experimental ones:
```suggestion
platforms = supported_platforms(; experimental=true)
```